Strategy11/formidable-forms
# Show/Hide some shortcodes contextually

Fix https://github.com/Strategy11/formidable-pro/issues/829

Related update: https://github.com/Strategy11/formidable-views/pull/592

### Test procedure
1. Create a form and go to its settings. Click on the 'Send email' action and try opening the shortcodes input `...` next to `To`, `From`, `CC` and `BCC` fields.
2. Click on `Advanced` tab in the shortcodes modal.
3. Confirm that `Admin email` and `Default from email` shortcodes are available in the list.
4. Now, go to the form builder and add fields that use the shortcodes modal, like an HTML field.
5. Go to the HTML field's options and try entering content into the `Content` field to open the shortcodes popup.
6. Confirm that `Default Msg`, `Default HTML`, `Default Plain` and `Form name` are not under the `Advanced` tab.

### Screen record
![CleanShot 2024-10-16 at 14 39 07](https://github.com/user-attachments/assets/19695b04-50c6-470b-8ccb-cb6e3d89525d)
